test/tools/spec-runner/index.js

```diff
@@ -169,10 +169,16 @@ function generateTopologyTests(testSuites, testContext, filter) {
 }
 
 function shouldRunSpecTest(configuration, requires, spec, filter) {
-  if (requires.authEnabled && process.env.AUTH !== 'auth') {
+  if (requires.authEnabled) {
     // TODO(NODE-3488): We do not have a way to determine if auth is enabled in our mocha metadata
     // We need to do a admin.command({getCmdLineOpts: 1}) if it errors (code=13) auth is on
-    return false;
+    if (process.env.AUTH !== 'auth') {
+      return false;
+    }
+
+    if (spec.failPoint.configureFailPoint === 'failCommand') {
+      return false;
+    }
```

Comment:
Why is this also gated within authEnabled? you should be able to run failCommands even when auth is turned on